core/app/models/spree/taxon.rb

```diff
@@ -196,15 +209,21 @@ def active_products
     end
 
     def regenerate_pretty_name_and_permalink
-      set_permalink
-      update_columns(pretty_name: generate_pretty_name, updated_at: Time.current)
+      if Spree.use_translations?
+        regenerate_translations_pretty_name_and_permalink
```

Comment:
This should not be needed theoretically because if we are using other language than the default, then `mobility` should update the model from the translation table and not the base one, so this `if` should always be `false`

Reply:
So we're using it because we're calling `update_columns` and this doesn't trigger updates from mobility?